.gitignore

```diff
@@ -6,3 +6,4 @@ polyfill/script.js.map
 polyfill/test262/
 out/
 .vscode/
+package-lock.json
```

Comment:
Given the precedent, I guess you should get rid of this and instead put this in your global gitignore? Or maybe we could just keep it for now and I could make a follow-up removing all source-independent entries from gitignore...

Reply:
package-lock shouldn't go in a global gitignore; if it's in a local one, then `package-lock=false` should also be in `.npmrc` to disable the lockfile for everyone.

Reply:
ok reverted in follow-on commit face42de.